backend/src/main/java/com/redhat/cloud/notifications/config/BackendConfig.java

```diff
@@ -106,9 +107,9 @@ void logConfigAtStartup(@Observes Startup event) {
         config.put(drawerToggle, isDrawerEnabled());
         config.put(EMAILS_ONLY_MODE, isEmailsOnlyModeEnabled());
         config.put(ERRATA_MIGRATION_BATCH_SIZE, getErrataMigrationBatchSize());
-        config.put(KESSEL_INVENTORY_ENABLED, isKesselInventoryEnabled());
-        config.put(KESSEL_INVENTORY_INTEGRATIONS_REMOVAL_ENABLED, areKesselInventoryIntegrationRemovalsEnabled());
-        config.put(KESSEL_RELATIONS_ENABLED, isKesselRelationsEnabled());
+        config.put(KESSEL_INVENTORY_ENABLED, isKesselInventoryEnabled(null));
+        config.put(KESSEL_INVENTORY_INTEGRATIONS_REMOVAL_ENABLED, areKesselInventoryIntegrationRemovalsEnabled(null));
+        config.put(KESSEL_RELATIONS_ENABLED, isKesselRelationsEnabled(null));
```

Comment:
I feel like we should maybe not even report these if we can't specify an accurate value at startup, what do you think?